javascript/javascript2/week3/js2week3.js

```diff
@@ -0,0 +1,167 @@
+const head = document.getElementById("header")
+
+head.innerHTML = "I am the header and I shall be JavaScript 2 week 3"
+
+
+// 1- Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+
+let textCalled = function () {
+    console.log("the text Called after 2.5");
+}
+
+setTimeout(textCalled, 2500);
+
+// 2- Create a function that takes 2 parameters: delay and stringToLog.
+// Calling this function should log out the stringToLog after delay seconds.
+// Call the function you have created with some different arguments.
+
+
+
+
+function logOutString (stringToLog, delay) {
+    setTimeout(() => {console.log(stringToLog)}, delay)
+};
+
+logOutString("This is a 5 seconds check", 5000)
+
+// 3- Create a button in html. 
+// When clicking this button, use the function you created in the previous task to log out the text:
+// Called after 5 seconds 5 seconds after the button is clicked.
+
+const btn = document.createElement("BUTTON");
+btn.innerHTML = "CLICK ME"; 
+btn.onclick = function logOutString (stringToLog, delay) {
+    setTimeout(() => {console.log("Called after 5 seconds")}, 5000)
+};
+document.body.appendChild(btn);
+
+// 4- Create two functions and assign them to two different variables. 
+// One function logs out Earth, the other function logs out Saturn.
+// Now create a new third function that has one parameter: planetLogFunction.
+// The only thing the third function should do is call the provided parameter function.
+// Try call the third function once with the Earth function and once with the Saturn function.
+
+let earthLogger = function earth () {
+    console.log("earth");
+};
+
+let saturnLogger = function saturn() {
+    console.log("saturn");
+}
+
+function planetLogFunction (planet) {
+    return planet
+};
+
+planetLogFunction(earthLogger())
+
+// 5- Create a button with the text called "Log location".
+// When this button is clicked the location (latitude, longitude) of the user should be logged out using this browser api
+
+const btnTow = document.createElement("BUTTON");
+btnTow.innerHTML = "Log location"; 
+document.body.appendChild(btnTow);
+
+
+
+  function success(pos) {
+    var crd = pos.coords;
+  
+    console.log('Your current position is:');
+    console.log(`Latitude : ${crd.latitude}`);
+    console.log(`Longitude: ${crd.longitude}`);
+    console.log(`More or less ${crd.accuracy} meters.`);
+  }
+  
+  
+  
+  btnTow.onclick = navigator.geolocation.getCurrentPosition(success);
```

Comment:
The current position will be displayed when the window is loaded in here. There is no meaning for the button you created here
If you want to get the position when you click the button then add the 'addEventListener' to the button element